tools/sense_studio/static/demos.js

```diff
@@ -0,0 +1,219 @@
+
+document.addEventListener("DOMContentLoaded", function () {
+    toggleMETConverters();
+});
+
+function toggleMETConverters() {
+    let demo = document.getElementById('demo');
+    let metToCalorieConverters = document.getElementById('metToCalorieConverters');
+
+    // Demos containing MET converters (e.g. weight, height, age, gender)
+    // If add new demo with any converter from above, add that script name here in below list (without .py)
+    let demosWithMETConverters = ["run_calorie_estimation", "run_fitness_tracker"];
+
+    if (demosWithMETConverters.includes(demo.value)) {
+        metToCalorieConverters.classList.remove('uk-hidden');
+    } else {
+        metToCalorieConverters.classList.add('uk-hidden');
+    }
+}
+
+async function getSupportedModelsByDemo(url) {
+    let demo = document.getElementById('demo');
+    let modelName = document.getElementById('modelName');
+    data = {
+        'demo': demo.value
+    };
+
+    let response = await asyncRequest(url, data);
+    modelName.innerHTML = null;
+    let models = response['models'];
+
+    if (models.length > 0) {
+        modelName.classList.remove('uk-form-danger');
+        for (model of models) {
+            modelName.insertAdjacentHTML('beforeend', `<option>${model}</option>`);
+        }
+    } else {
+        modelName.classList.add('uk-form-danger');
+        modelName.insertAdjacentHTML('beforeend', `<option value="">No models available</option>`);
+    }
+
+}
+
+function addTerminalMessage(message) {
+    let terminal = document.getElementById('demoTerminal');
+    terminal.insertAdjacentHTML('beforeend', `<div class='monospace-font'><b>${message}</b></div>`);
+    terminal.scrollTop = terminal.scrollHeight;
+}
+
+function streamDemo(message) {
+    let frame = document.getElementById('frame');
+    frame.src = message.image;
+    frame.width = message.width;
+    frame.height = message.height;
+}
+
+async function startDemo(url) {
+    let demo = document.getElementById('demo');
+    let weight = document.getElementById('weight').value;
+    let age = document.getElementById('age').value;
+    let height = document.getElementById('height').value;
+    let gender = document.getElementById('gender').value;
+    let modelName = document.getElementById('modelName').value;
+    let webcamInput = document.getElementsByName('inputSource')[0];
+    let saveVideo = document.getElementById('saveVideo');
+    let inputVideoPath = document.getElementById('inputVideoPath');
+    let inputVideoPathValue = (webcamInput.checked) ? '' : inputVideoPath.value;
+    let outputVideoName = document.getElementById('outputVideoName');
+    let outputVideoNameValue = (saveVideo.checked) ? outputVideoName.value : '';
+    let title = document.getElementById('title').value;
+    let buttonRunDemo = document.getElementById('btnRunDemo');
```

Comment:
Should we stick to the same variable names everywhere? i.e. Changing the element id to also be `buttonRunDemo` to make it easier to track the same reference?